Assets/MRTK/Providers/ObjectMeshObserver/Microsoft.MixedReality.Toolkit.Providers.ObjectMeshObserver.asmdef

```diff
@@ -6,9 +6,7 @@
         "Microsoft.MixedReality.Toolkit.Editor.Utilities"
     ],
     "optionalUnityReferences": [],
-    "includePlatforms": [
-        "Editor"
-    ],
+    "includePlatforms": [],
     "excludePlatforms": [],
```

Comment:
Attempted a Standalone build and there are errors associated with the ObjectMeshObserver Provider:
![image](https://user-images.githubusercontent.com/53493796/85315091-462dd600-b46f-11ea-91a7-5fd95b19060b.png)

```
Assets\MRTK\Providers\ObjectMeshObserver\RoomFileFormat\RoomFileImporter.cs(6,32): error CS0234: The type or namespace name 'AssetImporters' does not exist in the namespace 'UnityEditor.Experimental' (are you missing an assembly reference?)

```

```
Assets\MRTK\Providers\ObjectMeshObserver\SpatialObjectMeshObserverProfileInspector.cs(20,17): error CS0246: The type or namespace name 'SerializedProperty' could not be found (are you missing a using directive or an assembly reference?)
```

Reply:
Thanks for catching this! Should be fixed up in the latest now. It turns out yeah, it shouldn't be marked for all platforms because of this specific reason (though I suspect this could also be changed to work by not using editor APIs).